interface/weno/logsync.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+/*
+ *  @package OpenEMR
+ *  @link    http://www.open-emr.org
+ *  @author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  @copyright Copyright (c) 2021 Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+require_once("../globals.php");
+require_once($GLOBALS["srcdir"] . "/formatting.inc.php");
```

Comment:
formatting.inc.php is already automatically included whenever bring in globals.php, so can remove above line